test/integration/admin_login_test.rb

```diff
@@ -129,4 +128,24 @@ def setup
     click_button "Update"
     assert page.has_content?("Italian Drip")
   end
+
+  test "an admin can view all the orders" do
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    @non_admin = User.create(username: 'yayaya',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.